Firestore/Example/Tests/Integration/API/FIRQueryTests.m

```diff
@@ -111,6 +111,23 @@ - (void)testQueryWithFieldPaths {
   XCTAssertEqualObjects(FIRQuerySnapshotGetIDs(snapshot), (@[ @"b", @"a" ]));
 }
 
+- (void)testQueryWithPredicate {
```

Comment:
Please fix the trailing space travis failure - 

$ ! git grep -I ' $'
Firestore/Example/Tests/Integration/API/FIRQueryTests.m:  
The command "! git grep -I ' $'" exited with 1.

Reply:
done